Microsoft.Azure.Cosmos/src/Routing/GlobalEndpointManager.cs

```diff
@@ -173,14 +319,14 @@ public string GetLocation(Uri endpoint)
 
         public void MarkEndpointUnavailableForRead(Uri endpoint)
         {
-            DefaultTrace.TraceInformation("Marking endpoint {0} unavailable for read", endpoint);
+            DefaultTrace.TraceInformation(DateTime.UtcNow + "Marking endpoint {0} unavailable for read", endpoint);
```

Comment:
Doesn't the default trace already contains the time?

Reply:
nit: What about adding a space?
```suggestion
            DefaultTrace.TraceInformation("{1} Marking endpoint {0} unavailable for read", endpoint, DateTime.UtcNow);
```

Reply:
I added this while debugging. At least on the VS debug console output it does not. Reverted.